docker/build/assets.Dockerfile

```diff
@@ -129,10 +182,19 @@ RUN dnf install -y --nobest --setopt=install_weak_deps=False ${PYTHON}-devel &&
     ${PYTHON} -m pip install numpy build auditwheel patchelf
 
 RUN cd /cuda-quantum && source scripts/configure_build.sh && \
-    LLVM_INSTALL_PREFIX="$(mktemp -d)" && \
+    # Needed to retrigger the LLVM build, since the MLIR Python bindings
+    # are not built in the prereqs stage.
```

Comment:
Can we not rebuild LLVM again? In fact, let's just build it once per architecture and stash the binaries someplace. It doesn't need to be built multiple times.

Reply:
We are not rebuilding here, really. The prereqs stage builds everything except the python bindings. This build uses the same build folder and thus ends up only building the python bindings that are not yet built. I am merely deleting the install folder so the script doesn't skip over this. LLVM is only built twice (we have to build in two stages to eliminate any dependencies on an external compiler).